pkg/doc/signature/verifier/public_key_verifier.go

```diff
@@ -370,6 +372,40 @@ func NewECDSAES521SignatureVerifier() *ECDSASignatureVerifier {
 	}
 }
 
+// NewBBSG2SignatureVerifier creates a new BBSG2SignatureVerifier.
+func NewBBSG2SignatureVerifier() *BBSG2SignatureVerifier {
+	return &BBSG2SignatureVerifier{}
+}
+
+// BBSG2SignatureVerifier is a signature verifier that verifies a BBS+ Signature
+// taking Bls12381G2Key2020 public key bytes as input.
+// The reference implementation https://github.com/mattrglobal/bls12381-key-pair supports public key bytes only,
+// JWK is not supported. JWK is not supported as well here.
```

Comment:
Should be `JWK is not supported here as well` instead of `JWK is not supported as well here.`

`JWK is not supported as well here.` actually accidentally implies that JWK is supported, just to a lesser degree than something else.

Reply:
fixed